static/schemas/neon-api/v1/neon_gasPrice/response.json

```diff
@@ -0,0 +1,141 @@
+{
+  "type": "object",
+  "description": "Response to request for transaction gas price",
+  "required": [
+    "jsonrpc",
+    "id",
+    "result"
+  ],
+  "properties": {
+    "jsonrpc": {
+      "type": "string",
+      "default": "2.0",
+      "description": "The JSON RPC version"
+    },
+    "id": {
+      "type": "string",
+      "examples": [
+        "OpEgqw7r98qwerqwehkjhwe8"
+      ],
+      "description": "End-user provided identifier for the request"
+    },
+    "result": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "gas_price": {
+            "description": "Gas fee as hexadecimal value in Galan (NEON).",
```

Comment:
@afalaleev I need property descriptions (confirmed here) and for all the Results array below, please.

Reply:
Or I can rename suggested_gas_price to gas_price.

Reply:
This is a schema describing all possible properties that may be returned as a JSON object -- and `gas_price` is one of those properties as per the Notion page"
![image](https://github.com/neonlabsorg/neon-evm.docs/assets/20266893/1b4d7cc7-c11f-400d-8720-400e9c3042f5)
So, I am now very confused -- are you saying that `gas_price` does not exist?
I started this conversation asking for you to confirm the description (as I added the Galan detail)